packages/circuits/circom/circuits.json

```
@@ -29,5 +29,10 @@
         "file": "poseidon-cipher",
         "template": "PoseidonPerm",
         "params": [2]
+    },
+    "ecdh": {
+        "file": "ecdh",
+        "template": "Ecdh",
+        "params": []
```

Is `publicKey` a public signal? If yes, we should add it to the `pubs`.

```suggestion
        "file": "ecdh",
        "template": "Ecdh",
        "pubs": ["publicKey"],
        "params": []
```

---

I'm not sure here, it's never been used on its own. But it does make sense for it to be a public signal